09-minimal-dv-job.md

```diff
@@ -0,0 +1,155 @@
+allows you to access events stored in DSTs and copy the information to ROOT 
+ntuples.
+You tell DaVinci what you want it to do through _options files_, written in 
+Python.
+There are lots of things you can do with DaVinci options files, as there's lots 
+of information available on the DST, but for now we'll just work on getting the 
+bare essentials up and running.
+
+Our main tool will be the `DecayTreeTuple` object, which we'll create inside a 
+file we will call `ntuple_options.py`:
+
+```python
+from Configurables import DecayTreeTuple
+from DecayTreeTuple.Configuration import *
+
+# Stream and stripping line we want to use
+stream = 'CharmCompleteEvent'
+line = 'D2hhCompleteEventPromptDst2D2RSLine'
+
+# Create an ntuple to capture D*+ decays from the StrippingLine line
+dtt = DecayTreeTuple('TupleDstToD0pi_D0ToKpi')
+dtt.Inputs = ['/Event/{0}/Phys/{1}/Particles'.format(stream, line)]
+dtt.Decay = '[D*(2010)+ -> (D0 -> K- pi+) pi+]CC'
+```
+
+This imports the `DecayTreeTuple` class, and then creates an object called 
+`dtt` representing our ntuple.
+Once DaVinci has run, the resulting ntuple will be saved in a folder within the 
+output ROOT file called `TupleDstToD0pi_D0ToKpi`.
+
+The `Inputs` attribute specifies where `DecayTreeTuple` should look for 
+particles, and here we want it to look at the output of the stripping line 
+we're interested in.
+
+As stripping lines can save many decays to a DST, the `Decay` attribute 
+specifies what decay we would like to have in our ntuple.
+If there are no particles at the `Input` location, or the `Decay` string 
+doesn't match any particles at that location, the ntuple will not be filled.
+
+> ## Decay descriptors {.callout}
+>
+> There is a special syntax for the `Decay` attribute string, commonly called 
+> 'decay descriptors', that allow a lot of flexibility with what you accept.
+> For example, `D0 -> K- X+` will match any D0 decay that contains one 
+> negatively charged kaon and one positively charged track of any species.
+> More information the decay descriptor syntax can be found on the [LoKi decay 
+> finders TWiki 
+> page](https://twiki.cern.ch/twiki/bin/view/LHCb/FAQ/LoKiNewDecayFinders).
+
+Now we need to tell DaVinci how to behave.
+The `DaVinci` class allows you to tell DaVinci how many events to run over, 
+what type of data is being used, what algorithms to run over the events, and so 
+on.
+
+There are [many configuration 
+attributes](http://lhcb-release-area.web.cern.ch/LHCb-release-area/DOC/davinci/releases/v36r6/doxygen/py/dc/d2f/class_da_vinci_1_1_configuration_1_1_da_vinci.html#ac788f6a80f5f61d47056debe7b86ca71) 
+defined on the `DaVinci` object, but we will only set the ones that are 
+necessary for us.
+
+```python
+from Configurables import DaVinci
+
+# Configure DaVinci
+DaVinci().UserAlgorithms += [dtt]
+DaVinci().InputType = 'DST'
+DaVinci().TupleFile = 'DVntuple.root'
+DaVinci().PrintFreq = 1000
+DaVinci().DataType = '2011'
```

Comment:
It is. The way to know is to look at the BK path. Can we add that sentence somewhere?